src/Fixer/Whitespace/NoTrailingWhitespaceFixer.php

```diff
@@ -70,15 +70,6 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
```

Comment:
yep, it was added because of comment.
if removed, the comment needs to be added in every mentioned fixer.